masq_lib/src/multi_config.rs

```diff
@@ -88,6 +88,13 @@ impl<'a> MultiConfig<'a> {
             let message = format!("Invalid value: {}", &captures[2]);
             return ConfiguratorError::required(name, &message);
         }
+        let another_invalid_value_regex =
```

Comment:
If you removed the capture group you're not using in each of these cases, you might be able to combine them and use the regex as a parameter--either by making two calls to an outboard method or by putting the regexes in a vector and iterating through it.

Reply:
The first regex has `name` and `value` at positions `1` and `2` respectively. The second regex has them at places `2` and `1` respectively.

So, I need to introduce `index_of_name` and `index_of_value` inside the outboard function and it looks something like this:

```rust
    fn check_for_inavlid_value_err(
        error_msg: &str,
        pattern: Regex,
        index_of_name: usize,
        index_of_value: usize,
    ) -> Option<ConfiguratorError> {
        if let Some(captures) = pattern.captures(error_msg) {
            let name = &captures[index_of_name];
            let message = format!("Invalid value: {}", &captures[index_of_value]);

            Some(ConfiguratorError::required(name, message))
        }

        None
    }
```


Due to the irregularity and low number of cases (only 2), it turns out that such a functionality won't help us minimise the number of lines of the parent function.